rust/crd/Cargo.toml

```
@@ -5,14 +5,13 @@ edition = "2018"
 license = "OSL-3.0"
 name = "stackable-opa-crd"
 repository = "https://github.com/stackabletech/opa-operator"
-version = "0.4.0-nightly"
+version = "0.3.0-nightly"
```

This doesn't look right.
```suggestion
version = "0.4.0-nightly"
```